docs/resiliency/failure-mode-analysis.md

```diff
@@ -383,6 +383,13 @@ For more information, see [Overview of Service Bus dead-letter queues][sb-dead-l
 
 **Diagnostics**. Use [Azure Activity Logs][azure-activity-logs].
 
+### Multiple VM instances across multiple tiers become unavailable or unhealthy.
+**Detection**. Configure Azure traffic manager to monitor public endpoint availability. Also, configure a Load Balancer [health probe][lb-probe] that signals whether the VM instance is healthy. The probe should check whether critical functions are responding correctly.
```

Comment:
Are we recommending that every IaaS deployment should use traffic manager to monitor availability?